index.js

```diff
@@ -573,6 +573,19 @@ const anyObject = {
   }
 }
 
+const anyDate = {
+  preencode (state, o) {
+    utf8.preencode(state, o.toISOString())
+  },
+  encode (state, o) {
+    utf8.encode(state, o.toISOString())
+  },
+  decode (state) {
+    const isoString = utf8.decode(state)
+    return new Date(isoString)
```

Comment:
lets encode as uint, seems much smaller

Reply:
ie o.getTime() and new Date(ms)

Reply:
did it for you in 2.16.0 \o/

Reply:
Agreed, `unit` for ms is much smaller and better.